Source/MethodCall.cs

```diff
@@ -81,42 +81,36 @@ public IVerifies Raises(Action<TMock> eventExpression, params object[] args)
 		}
 	}
 
-	internal class TypeEqualityComparer : IEqualityComparer<Type>
-	{
-		public bool Equals(Type x, Type y)
-		{
-			return y.IsAssignableFrom(x);
-		}
-
-		public int GetHashCode(Type obj)
-		{
-			return obj.GetHashCode();
-		}
-	}
-
 	internal partial class MethodCall : IProxyCall, ICallbackResult, IVerifies, IThrowsResult
 	{
-		// Internal for AsMockExtensions
-		private Expression originalExpression;
-		private Exception thrownException;
-		private Action<object[]> setupCallback;
-		private List<IMatcher> argumentMatchers = new List<IMatcher>();
-		private EventInfo mockEvent;
-		private Delegate mockEventArgsFunc;
-		private object[] mockEventArgsParams;
```

Comment:
These 3 event fields get replaced by a single one, `raiseEventResponse`. This means that for all setups who do *not* raise some event (which is probably most of them), we save two fields. (And the few setups that *do* raise some event will incur one additional allocation.)